hedera-node/pom.xml

```diff
@@ -265,6 +265,10 @@
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.datatype</groupId>
+      <artifactId>jackson-datatype-jsr310</artifactId>
+    </dependency>
```

Comment:
Does test-clients or other module need this? If not, just add a dependency here, and remove the one in root pom file.